features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/webpack.config.js

```diff
@@ -29,6 +29,9 @@ const config = {
         chunkFilename: '[name].bundle.js',
         publicPath: 'site/public/dist/',
     },
+    node: {
```

Comment:
Do we need to mock the node's `fs` module as an empty object here ?

https://webpack.js.org/configuration/node/#node

Reply:
This is to fix a build failure in integrating with the Swagger Editor.

https://github.com/webpack-contrib/css-loader/issues/447

Reply:
Ack, Thanks for the information :+1: